From 13a92bfbd534d9f237f38b048be2740c26f6749e Mon Sep 17 00:00:00 2001
From: David Pacheco <dap@joyent.com>
Date: Tue, 20 Jun 2017 16:28:27 -0700
Subject: [PATCH] MANTA-3317 upload-mako-ls uses all memory, then uploads
 incomplete dump Reviewed by: Brian Bennett <brian.bennett@joyent.com>
 Reviewed by: Kody A Kantor <kody.kantor@gmail.com> Approved by: Brian Bennett
 <brian.bennett@joyent.com> Approved by: Kody A Kantor <kody.kantor@gmail.com>

---
 bin/upload_mako_ls.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bin/upload_mako_ls.sh b/bin/upload_mako_ls.sh
index 92b4582..c3c5eee 100755
--- a/bin/upload_mako_ls.sh
+++ b/bin/upload_mako_ls.sh
@@ -33,7 +33,7 @@ DIR_TYPE='application/json; type=directory'
 LOG_TYPE='application/x-bzip2'
 PID=$$
 PID_FILE=/tmp/upload_mako_ls.pid
-TMP_DIR=/tmp/mako_dir
+TMP_DIR=/var/tmp/mako_dir
 LISTING_FILE=$TMP_DIR/$MANTA_STORAGE_ID
 MANTA_DIR=/mako
 
-- 
2.21.0

